scipy/sparse/_generate_sparsetools.py

```diff
@@ -419,12 +419,15 @@ def main():
     # Generate code for method struct
     method_defs = ""
     for name in names:
-        method_defs += f"NPY_VISIBILITY_HIDDEN PyObject *{name}_method(PyObject *, PyObject *);\n"
+        method_defs += (f"NPY_VISIBILITY_HIDDEN PyObject *{name}"
+                        f"_method(PyObject *, PyObject *);\n")
 
     method_struct = """\nstatic struct PyMethodDef sparsetools_methods[] = {"""
     for name in names:
-        method_struct += """
-        {{"{name}", (PyCFunction){name}_method, METH_VARARGS, NULL}},""".format(**dict(name=name))
+        method_struct += ("""
+            {{"{name}", (PyCFunction){name}_method, METH_VARARGS, NULL}},"""
+            .format(**dict(name=name))
+        )
```

Comment:
I'm suspicious of these changes as the `Compiling C++ object scipy/sparse/sparsetools/_sparsetools.cpython-311-x86_64-linux-gnu.so.p/csr.cxx.o` step of `python dev.py build` seems to be taking abnormally long.